fcl/dunefdvd/detsim/workflow_detsim_dunevd10kt.fcl

```diff
@@ -37,15 +37,15 @@ dunefd_vertdrift_tpc:
 dunefd_vertdrift_pds_aronly:
 [
     sipmArOnly,
-    sipmArOnlyExt,
-    opdigiArOnly
+    # sipmArOnlyExt,
+    # opdigiArOnly
 ]
 
 dunefd_vertdrift_pds_xeonly:
 [
     sipmXeOnly,
-    sipmXeOnlyExt,
-    opdigiXeOnly
+    # sipmXeOnlyExt,
+    # opdigiXeOnly
```

Comment:
Does this run?  I'd have guessed that having `sipmArOnly, ]` (a comma followed by a bracket) would fail to be read by fcl.